tools/ssm/linux-amd64-deb/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Use `#!/usr/bin/env bash` https://github.com/aws/amazon-cloudwatch-agent/blob/53f8c7cd6c2e1977bc45ea55f70298b0d7dce942/Tools/src/create_darwin.sh#L1 or `sh` https://github.com/aws/amazon-cloudwatch-agent/blob/53f8c7cd6c2e1977bc45ea55f70298b0d7dce942/packaging/dependencies/amazon-cloudwatch-agent-ctl#L1

Though I think all of them should work on most vms and bms.

Reply:
Agree. I would prefer to `#!/usr/bin/env bash` and verify in all supported AMIs.

Reply:
Fixed